### DISCUSSION TITLE
Add a display of a roofs for building

### DISCUSSION THREAD

**Discussion by superyasha2** [organicmaps/organicmaps#2202] [Ideas]
Add a rendering of the roofs, which are marks with the _[roof:shape](https://wiki.openstreetmap.org/wiki/Key:roof:shape)_ tag. Also _[roof:levels](https://wiki.openstreetmap.org/wiki/Key:roof:levels)_ tag could be used to determine the roof height because it can be different, like _[building:levels](https://wiki.openstreetmap.org/wiki/Key:building:levels)_ tag is used for marking building levels without roof and sets the height.

And there are some problems. Some roof shapes have a double meaning. _roof:shape=saltbox_ can be interpreted in different ways:
![изображение](https://user-images.githubusercontent.com/40822688/156174457-dc9bb32e-c964-4640-b354-ee864d508ee4.png)
I think you should display this roof type as on the second option because most of the edits to mark the roof type are made with StreetComplete (not sure), which shows the second form. By the way F4Map doesn't render this roof shape at all.

**Comment by biodranik** [comment]
Thanks! I've created an issue here: https://github.com/organicmaps/organicmaps/issues/2204
